ert_shared/_doc_utils/ert_jobs.py

```diff
@@ -185,18 +190,35 @@ def _generate_job_documentation(self, jobs, section_id, title):
             category_section_node = _create_section_with_title(
                 section_id=category + "-category", title=category.capitalize()
             )
+            job_sub_categories = job_categories[category]
+            print("Start", category)
+            for subcategory_index, subcategory in enumerate(
```

Comment:
I'm suspecting all of this code would read nicer if you are able to abbreviate the two variable names